### PR TITLE
docs(bin/setup): Add an example for an alias with multiple recipients

### PR DESCRIPTION
**Pull request by jsonn**
Small documentation improvement as there are different common mechanisms for this kind of action.

### CODE REVIEW

File: target/bin/addalias [hunk 1]
```diff
@@ -33,6 +33,9 @@ ${ORANGE}EXAMPLES${RESET}
     ${LWHITE}./setup.sh alias add alias@example.com recipient@example.com${RESET}
         Add the alias 'alias@example.com' for the mail account 'recipient@example.com'.
 
+    ${LWHITE}./setup.sh alias add alias2@example.com "tgt1@example.com, tgt2@example.com"${RESET}
```
Comment (polarathene) [inline target/bin/addalias:36]:
```suggestion
    ${LWHITE}./setup.sh alias add alias@example.com "recipient@example.com, another-recipient@example.com"${RESET}
```

**Comment by jsonn** [comment]
Works for me

**Review by georglauterbach — DISMISSED** [review]

File: target/bin/addalias [hunk 2]
```diff
@@ -33,6 +33,9 @@ ${ORANGE}EXAMPLES${RESET}
     ${LWHITE}./setup.sh alias add alias@example.com recipient@example.com${RESET}
         Add the alias 'alias@example.com' for the mail account 'recipient@example.com'.
 
+    ${LWHITE}./setup.sh alias add alias@example.com "recipient@example.com, another-recipient@example.com"${RESET}
```
Comment (polarathene) [inline target/bin/addalias:36]:
```suggestion
    ${LWHITE}./setup.sh alias add alias@example.com 'recipient@example.com, another-recipient@example.com'${RESET}
```

**Review by polarathene — APPROVED** [review]
LGTM 👍 

Thanks for the contribution! ❤️

**Pull request merged**